### PR TITLE
(fix) Updated YAML's in deploy folder

### PR DESCRIPTION
**Pull request by rahulchheda**
Signed-off-by: Rahul M Chheda <rahul.chheda@mayadata.io>

**What this PR does / why we need it**:
   
    - Fixed approriate YAML's in deploy folder

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
    - https://github.com/litmuschaos/litmus/issues/1393
**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

**Comment by ksatchit** [comment]
@rahulchheda can we sanitize the deploy folder: 

- Add the corresponding/mapped example in chaosexperiment CR ? i.e., the existing one seems to be pod-delete. We can either choose to go w/ pod-delete for both exp/engine or network-loss for both. 
- Probably worth including a sample rbac yaml 
- A README explaining very briefly on what is the recommended order of yaml application followed by a reference to litmus-docs for more information --- would complete the set?

**Comment by ksatchit** [comment]
An alternative approach is to keep CRDs here and just put out a readme asking users to refer litmus-docs :)

**Comment by ksatchit** [comment]
It would be great to index the deploy files for quick user evaluation. Say: 

01-chaos-operator.yaml (w/ `latest` or `ci` tags)
02-nginx-deploy.yaml (an annotated one w/ `litmuschaos.io.chaos: true`)
03-pod-delete-chaosexperiment.yaml 
04-pod-delete-rbac.yaml
05-chaosengine.yaml (w/o overrides, simplistic version)

The readme can include application of these and steps to view results/engine status

File: deploy/02-annotated-nginx-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx
+        imagePullPolicy: Always
+        name: nginx
+        resources: {}
```
Comment (ksatchit) [inline deploy/02-annotated-nginx-deploy.yaml:26]: @rahulchheda can we keep only the sample yaml spec w/o creationTimeStamp, resources, terminationMessagePath/Policy/SchedulerName etc.,

Comment (rahulchheda) [reply deploy/02-annotated-nginx-deploy.yaml:26]: Sure.

File: deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+        - Apply Nginx Deployment: `kubectl apply -f 02-annotated-nginx-deploy.yaml`
+
+        - Apply Pod Delete Chaos Experiment: `kubectl apply -f 03-pod-delete-experiment.yaml`
+
+        - Apply RBAC Manifest for Chaos Engine: `kubectl apply -f 05-chaos-engine.yaml`
+        
+        - At last, apply the ChaosEngine: `kubectl apply -f 04-rbac-manifest.yaml`
+        
+        - To check the status of the this experiment, use this command
```
Comment (chandankumar4) [inline deploy/README.md:21]: fix the typo in this line also insteat of -oyaml describe will be better

File: deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        
+          `kubectl get chaosengine engine -n litmus -oyaml`
+
+            And check the status of variou experiments executed.
```
Comment (chandankumar4) [inline deploy/README.md:25]: some expected output if you add it will be better

**Pull request merged**